File: uri 1045.py
l = list(map(float, input().split(" ")))
l.sort(reverse=True)
A, B, C = l
if A >= B+C:
    print("NAO FORMA TRIANGULO")
else:
    if A**2 == (B**2 + C**2):
        print("TRIANGULO RETANGULO")
    if A**2 > (B**2 + C**2):
        print("TRIANGULO OBTUSANGULO")
    if A**2 < (B**2 + C**2):
        print("TRIANGULO ACUTANGULO")
    if A == B == C:
        print("TRIANGULO EQUILATERO")
    if A == B or B == C or C == A:
        if A != B or B != C or C != A:
            print("TRIANGULO ISOSCELES")

File: uri 1045.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    double a[3],k,A,B,C;
    int i,j;
    scanf("%lf %lf %lf",&a[0],&a[1],&a[2]);
    for(i=0;i<2;i++){
        for(j=0;j<2;j++){
            if(a[j]<a[j+1]){
                k=a[j+1];
                a[j+1]=a[j];
                a[j]=k;
            }
        }
    }
    A=a[0];
    B=a[1];
    C=a[2];
    if(A>=(B+C))
        printf("NAO FORMA TRIANGULO\n");

    else{
        if((A*A)==((B*B)+(C*C)))
            printf("TRIANGULO RETANGULO\n");
        if((A*A)>((B*B)+(C*C)))
            printf("TRIANGULO OBTUSANGULO\n");
        if((A*A)<((B*B)+(C*C)))
            printf("TRIANGULO ACUTANGULO\n");
        if(A==B && B==C)
            printf("TRIANGULO EQUILATERO\n");
        if(A==B || B==C || C==A){
            if(A!=B || B!=C || C!=A)
                printf("TRIANGULO ISOSCELES\n");
        }
    }
    return 0;
}
